src/containers/user-profile/comments-with-rating-block/CommentsWithRatingBlock.tsx

```diff
@@ -44,10 +38,21 @@ const CommentsWithRatingBlock = ({
       ? 'userProfilePage.reviews.titleTutor'
       : 'userProfilePage.reviews.titleStudent'
 
-  const items: MockReview[] =
-    userRole === UserRoleEnum.Tutor
-      ? [...responseMock.items]
-      : [...responseMockStudents.items]
+  const getReviews = useCallback(
+    () => ReviewService.getUserReviews({ userId: userId, userRole: userRole }),
+    [userId, userRole]
+  )
+
+  const { data, isLoading } = useQuery<ReviewsResponse>({
+    queryFn: getReviews,
+    queryKey: [['reviews', userId, userRole]],
```

Comment:
Maybe second array is not needed
```suggestion
    queryKey: ['reviews', userId, userRole],
```